Makefile

```diff
@@ -15,9 +15,12 @@ goteleinfo: $(SRC)
 test: $(SRC_TESTS)
 	go test -v $(TEST_DIRS)
 
+arm: $(SRC)
```

Comment:
This should be replaced with something simpler like mentioning in the README file that cross-compiling can be achieved using the following command:

```shell
$ GOARCH=arm make
```